src/Data/Uniques/body.lua

```diff
@@ -1183,27 +1182,23 @@ Recover 50 Energy Shield when your Trap is triggered by an Enemy
 Victario's Influence
 Lacquered Garb
 Variant: Pre 2.6.0
-Variant: Pre 3.16.0
 Variant: Current
-Requires Level 52, 76 Dex, 76 Int
+Implicits: 0
 +1 to Level of Socketed Aura Gems
-{variant:1}Socketed Gems have 40% increased Mana Reservation Efficiency
-{variant:2}Socketed Gems have 60% increased Mana Reservation Efficiency
-{variant:3}Socketed Gems have 45% increased Reservation Efficiency
-{variant:1}Socketed Gems are Supported by level 1 Generosity
-{variant:2,3}Socketed Gems are Supported by level 30 Generosity
+{variant:1}Socketed Gems are Supported by Level 1 Generosity
+{variant:2}Socketed Gems are Supported by Level 30 Generosity
+Socketed Gems have 45% increased Reservation Efficiency
```

Comment:
20% and 30% versions of this stat still exist on standard.  I'm not sure why the variants were 40 and 60.

https://www.pathofexile.com/trade/search/Standard/m7OrKqei6

Reply:
The displayed reservation efficiency value on legacy Victario's Influence is cosmetic only. GGG forced this modifier to always grant 45% increased reservation efficiency in 3.16 regardless of the stat displayed.

Reply:
Ah, I remember that now.  Really curious how that works on their backend, then.  Ninja edit:
A special support:
![image](https://user-images.githubusercontent.com/1209372/183338987-527dc45b-ff9e-493b-a182-a8bf45100192.png)